canonical-web-and-design/canonical.com
# Add partner portal to the /partners seconary nav

## Done

- Add partner portal to the /partners seconary nav
- Moved the partner suru strip css into the p-strip scss file

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/partners
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that 'Partner portal' is in the seconary nav


## Issue / Card

Fixes #239

## Screenshots

![image](https://user-images.githubusercontent.com/441217/83330919-66acab00-a28a-11ea-976a-3a0632136bcb.png)
